engine/table/src/main/java/io/deephaven/engine/util/PickledResult.java

```diff
@@ -8,6 +8,7 @@
 /**
  * Pickled result for a Python fetch.
  */
+@Deprecated
```

Comment:
Could you leave the change out? I moved it in #2582?

Reply:
Removed - that said, it would have resolved cleanly with you move.